core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -35,6 +35,7 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
+import com.google.census.CensusContextFactory;
```

Comment:
The Census API is meant to be a general-purpose stats/tracing API, for which you can have your own implementation. The actual hooking mechanism is not there yet, but @dinooliva from Census team is working on it. At this point I don't see much benefit from introducing yet another stats API for gRPC, which will be more gRPC-specific but won't be more powerful than Census API. Do you have more specific reasons?